docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,88 @@
+#Share your Gutenberg Block with the World
+So you’ve created an awesome block have you? Care to share? Here is some basic information on how to submit a block to the Block Directory.
+
+Need help building a block? Check out our block tutorials to get started. 
```

Comment:
I would directly link to the documentation for block creation: https://github.com/WordPress/gutenberg/tree/master/docs/designers-developers/developers/tutorials/create-block

Reply:
I actually removed this line with the expectation that this document will tie closely into the `Finishing Touches` document and therefore may not be necessary. https://github.com/WordPress/gutenberg/issues/23484.

I'll add it back in later if we think it necessary.